### ISSUE TITLE
Remove montoring configuration from database / server configuration

### ISSUE THREAD

**Issue by per-samuelsson** [Starcounter/Starcounter#2980]
This part:

```xml
    <Monitoring>
      <MaxRestartNumber xsi:nil="true" />
      <ResetFailureCountPeriod xsi:nil="true" />
      <GracePeriodAfterConnectionLost xsi:nil="true" />
      <StartupType>Manual</StartupType>
      <MonitoringType>Disabled</MonitoringType>
    </Monitoring>
```

We never consume it, and its not supported in 2.x. Its a legacy, and it's unlikely that we'll start to support it in 2.x.

**Comment by dan31** [comment]
@per-samuelsson, that's actually an important question. You remember the "watchdog" discussion? We want that when something dies, it is restored. So that everything is "self-fault-tolerant". I think our system service doesn't work like that today. E.g. you can kill the gateway and no one will restore it, right? For production-use, we need some watch-dog behaviour. Very simple watch-dog-alike thing in Windows is having services auto-restart, but then one has to have services for all SC components. Also @bigwad has some opinion on that. I think @eriohl also have some opinion.

**Comment by eriohl** [comment]
I'm not all that familiar with how 2.x service installation operates. But this configuration I think is not for the service but for the "server/admin" I think? If it is not used it should be removed.

**Comment by per-samuelsson** [comment]
@dan31 

Yes, we certainly need that kind of functionality, but this configuration is a legacy, and we should drop it since it's basically nothing more than clutter today. 

In some old, old version, we had a trivial restarting of code hosts that unexpectedly crashed, but that code is based on a completely different arch. The configuration is a leftover from that.

I'm dropping this, and then lets open up an issue about monitoring separatly, ok? What we have now in terms of that, is that the admin server library do some monitoring of code host processes when they boot a new application, and can react to an error there (and restart the host with all functional applications, if injecting a new one failed). But thats basically it. We need a more throughout solution obv. But please don't mix that with this issue.

**Comment by dan31** [comment]
Sure @per-samuelsson, thanks for explaining! Let's keep it in some other issue. Maybe it exists somewhere and you will identify it during your cleanup.
